server/src/test/java/io/druid/server/AsyncManagementForwardingServletTest.java

```diff
@@ -436,7 +436,7 @@ public String getCurrentLeader()
       JettyServerInitUtils.addExtensionFilters(root, injector);
 
       final HandlerList handlerList = new HandlerList();
-      handlerList.setHandlers(new Handler[]{JettyServerInitUtils.wrapWithDefaultGzipHandler(root)});
+      handlerList.setHandlers(new Handler[]{JettyServerInitUtils.wrapWithDefaultGzipHandler(root, 4096, -1)});
```

Comment:
Should have used `ServerConfig.DEFAULT_GZIP_INFLATE_BUFFER_SIZE` and `Deflater.DEFAULT_COMPRESSION` instead of magic constants